### DISCUSSION TITLE
Using the program without the GUI

### DISCUSSION THREAD

**Discussion by parrothead81** [Maplespe/DWMBlurGlass#245] [General]
How do I apply Aero without running the GUI?

I'm asking because the Readme file says the program is fully open source except the GUI which requires proprietary UI software, but that it can be used without the GUI.

**Comment by parrothead81** [comment]
I've started a mockup of the GUI. It's functional but ugly (it's AutoHotKey) and none of the buttons do anything useful since it's a mockup. Feel free to use

https://github.com/parrothead81/dwmblurglassahkgui

**Reply by indigofairyx** [reply]
simple and pretty!
